doc/input-output-reference/src/overview/group-thermal-zone-description-geometry.tex

```diff
@@ -4464,6 +4464,15 @@ \subsubsection{Inputs}\label{inputs-32-001}
 
 \emph{OnIfHighZoneAirTempAndHighHorizontalSolar:} Shading is on if the zone air temperature exceeds the Setpoint (C) and if the horizontal solar radiation exceeds SetPoint 2 (W/m\(^{2}\)).
 
+The following three control types can be used to model human behavior of turning on shading during the day to reduce view luminance or solar gain and leaving shading on until a certain time point. They can only be used in zones with daylighting controls. Window view luminance at the first daylighting reference point is used to compare with luminance setpoint. They are applicable to any Shading Type except ExteriorScreen.
+
+\emph{OnIfHighLuminanceOrHighSolarTillMidnight:} Shading is on if the solar radiation incident on the window exceeds SetPoint (W/m\(^{2}\)) or the luminance from the window exceeds SetPoint 2 (cd/m\(^{2}\)). When shading is on, it remains on till midnight.
```

Comment:
@mjwitte and others, a question about naming and setpoint order contention: currently the new options I added are named "On if A Or B Till XX", and they all use two setpoints under WindowShadingControl, where setpoint is for B and setpoint2 is for A. i.e. the sequence of A and B are not aligned between option name and setpoint order. Is this okay or do we want to make sure they align, by either changing naming or changing setpoint sequence?

Reply:
These controls have become so complicated, I'm not sure if the order of the setpoints matter, but I suppose it would be best to align them, so I would vote for changing the control type names.

Reply:
Thanks. Names aligned.